samples/sampler/stories/default/Picker.js

```diff
@@ -1,51 +1,63 @@
-// import {storiesOf} from '@storybook/react';
-//
-// import Picker from '@enact/moonstone/Picker';
-//
-// import {decrementIcons, incrementIcons} from './icons';
-//
-// // Set up some defaults for info and knobs
-// const prop = {
-// 	orientation: ['horizontal', 'vertical'],
-// 	width: [null, 'small', 'medium', 'large']
-// };
-//
-// const airports = [
-// 	'San Francisco Airport Terminal Gate 1',
-// 	'Boston Airport Terminal Gate 2',
-// 	'Tokyo Airport Terminal Gate 3',
-// 	'נמל התעופה בן גוריון טרמינל הבינלאומי'
-// ];
-//
-// // const Config = mergeComponentMetadata('FormCheckboxItem', FormCheckboxItem);
-// Picker.displayName = 'Picker';
-//
-// storiesOf('Moonstone', module)
-// 	.add(
-// 		'Picker',
-// 		() => (
-// 			<Picker
-// 				aria-label={text('aria-label', Picker, '')}
-// 				decrementAriaLabel={text('decrementAriaLabel', Picker, '')}
-// 				decrementIcon={select('decrementIcon', ['', ...decrementIcons], Picker)}
-// 				disabled={boolean('disabled', Picker)}
-// 				incrementAriaLabel={text('incrementAriaLabel', Picker, '')}
-// 				incrementIcon={select('incrementIcon', ['', ...incrementIcons], Picker)}
-// 				joined={boolean('joined', Picker)}
-// 				noAnimation={boolean('noAnimation', Picker)}
-// 				onChange={action('onChange')}
-// 				orientation={select('orientation', prop.orientation, Picker, prop.orientation[0])}
-// 				width={select('width', prop.width, Picker, prop.width[3])}
-// 				wrap={boolean('wrap', Picker)}
-// 			>
-// 				{airports}
-// 			</Picker>
-// 		),
-// 		{
-// 			info: {
-// 				text: 'Basic usage of Picker'
-// 			}
-// 		}
-// 	);
+import Picker from '@enact/moonstone/Picker';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+
+import {decrementIcons, incrementIcons} from '../helper/icons';
+
+// Set up some defaults for info and controls
+const prop = {
+	orientation: ['horizontal', 'vertical'],
+	width: [null, 'small', 'medium', 'large']
+};
+
+const airports = [
+	'San Francisco Airport Terminal Gate 1',
+	'Boston Airport Terminal Gate 2',
+	'Tokyo Airport Terminal Gate 3',
+	'נמל התעופה בן גוריון טרמינל הבינלאומי'
+];
+
+Picker.displayName = 'Picker';
+
+export default {
+	title: 'Moonstone/Picker',
+	component: 'Picker'
+};
+
+export const _Picker = (args) => (
+	<Picker
+		aria-label={args['aria-label']}
+		decrementAriaLabel={args['decrementAriaLabel']}
+		decrementIcon={args['decrementIcon']}
+		disabled={args['disabled']}
+		incrementAriaLabel={args['incrementAriaLabel']}
+		incrementIcon={args['incrementIcon']}
+		joined={args['joined']}
+		noAnimation={args['noAnimation']}
+		onChange={action('onChange')}
+		orientation={args['orientation']}
+		width={args['width']}
+		wrap={args['wrap']}
+	>
+		{airports}
+	</Picker>
+);
+
+boolean('disabled', _Picker, Picker);
+boolean('joined', _Picker, Picker);
+boolean('noAnimation', _Picker, Picker);
+boolean('wrap', Picker);
```

Comment:
This controls is missing the story name and the control is not showing up
```suggestion
boolean('wrap', _Picker, Picker);
```